nested_EnsureClusterUserAssignedHasRbacToManageVMSS.bicep

```diff
@@ -0,0 +1,12 @@
+param kubeletidentityObjectId string
+
+var virtualMachineContributorRole = '${subscription().id}/providers/Microsoft.Authorization/roleDefinitions/9980e02c-c2be-4d73-94e8-173b1dc7cf3c'
+
+resource id 'Microsoft.Authorization/roleAssignments@2020-10-01-preview' = {
+  name: guid(resourceGroup().id)
```

Comment:
I know the guid WAS this in the past, but can we now make it a combination of resourceGroup().id and  `virtualMachineContributorRole` -- I think it was just wrong before how it was.  Having it just be the RG.id doesn't really make this assignment unique enough from a name.